Book-A-desk.Domain.Tests/ReservationsCommandHandlerUsingInstructionsTests.fs

```diff
@@ -0,0 +1,88 @@
+﻿namespace Book_A_desk.Domain.Tests
+
+open System
+open Book_A_Desk.Domain
+open Book_A_Desk.Domain.CommandHandler
+open Book_A_Desk.Domain.Office.Domain
+open Book_A_Desk.Domain.Reservation
+open Book_A_Desk.Domain.Reservation.Commands
+open Book_A_Desk.Domain.Reservation.Commands.Instructions.ReservationInstructions
+open Xunit
+
+#nowarn "40"
+module rec ReservationsCommandHandlerUsingInstructionsTests =
```

Comment:
I added a test to show what tests look like with an interpreter. They're kind of clunky since we need to implement a whole interpreter. I'm pretty certain though that we could make it very generic and reusable across the production code and tests.